main.go

```diff
@@ -94,16 +94,17 @@ func getCommitHash(gci *model.GitCommitInfo) int {
 	return headMax
 }
 
-func getReflogHash(gci *model.GitCommitInfo) {
+func getReflogHash() ([]string, error) {
+	var ReflogHashList []string
 	out, err := exec.Command("git", "reflog", "--format=%h").Output()
 	if err != nil {
```

Comment:
Since exception handling is done also at the caller of function.  You can choose following
- If you would handle error in `getReflogHash `, return ReflogHashList only.
- If you would NOT handle error in `getReflogHash `, return ReflogHashList and err object

Reply:
I choose the latter because it is easy to implement the unit test.